Avoid to copy manual at runtime as it results in initialization failure
https://github.com/sei40kr/dotfiles/blob/master/overlays/zinit/no-doc-copy.patch
--- a/zinit.zsh
+++ b/zinit.zsh
@@ -1306,16 +1306,6 @@ builtin setopt noaliases
         # Also set up */bin and ZPFX in general.
         command mkdir 2>/dev/null -p $ZPFX/bin
     }
-    [[ ! -d ${~ZINIT[MAN_DIR]}/man9 ]] && {
-        # Create ZINIT[MAN_DIR]/man{1..9}
-        command mkdir 2>/dev/null -p ${~ZINIT[MAN_DIR]}/man{1..9}
-    }
-    # Copy Zinit manpage so that man is able to find it
-    [[ ! -f $ZINIT[MAN_DIR]/man1/zinit.1 || \
-        $ZINIT[MAN_DIR]/man1/zinit.1 -ot $ZINIT[BIN_DIR]/doc/zinit.1 ]] && {
-        command mkdir -p $ZINIT[MAN_DIR]/man1
-        command cp -f $ZINIT[BIN_DIR]/doc/zinit.1 $ZINIT[MAN_DIR]/man1
-    }
 } # ]]]
 # FUNCTION: .zinit-load-object [[[
 .zinit-load-object() {
@@ -2886,7 +2876,7 @@ You can try to prepend {apo}${___q}{lhi}@{apo}'{error} to the ID if the last ice
            ;;

        (man)
-           man "${ZINIT[BIN_DIR]}/doc/zinit.1"
+           man zinit
            ;;
        (env-whitelist)
             shift
